.circleci/config.yml

```diff
@@ -1,17 +1,31 @@
-version: 2
+# Cache key for CircleCI. We want to invalidate the cache whenever the Bazel workspace or the
+# NPM dependencies changed.
+var_1: &cache_key angular-tsickle-{{ checksum "WORKSPACE" }}-{{ checksum "yarn.lock" }}
+var_2: &default_docker_image l.gcr.io/google/bazel:0.18.0
```

Comment:
What's the version number here? is this another place that we have to bump the version when we move to a newer bazel?

Can you add a comment?

Reply:
Done.